napari/layers/image/image.py

```diff
@@ -633,20 +633,21 @@ class Image(IntensityVisualizationMixin, _ImageBase):
     blending : str
         One of a list of preset blending modes that determines how RGB and
         alpha values of the layer visual get mixed. Allowed values are
-        {'opaque', 'translucent', and 'additive'}.
+        {'translucent', 'translucent_no_depth', 'additive', 'minimum', 'opaque'}.
     cache : bool
         Whether slices of out-of-core datasets should be cached upon retrieval.
         Currently, this only applies to dask arrays.
     colormap : str, napari.utils.Colormap, tuple, dict
-        Colormap to use for luminance images. If a string must be the name
-        of a supported colormap from vispy or matplotlib. If a tuple the
-        first value must be a string to assign as a name to a colormap and
-        the second item must be a Colormap. If a dict the key must be a
-        string to assign as a name to a colormap and the value must be a
+        Colormaps to use for luminance images. If a string, it can be the name
+        of a supported colormap from vispy or matplotlib or the name of
+        a vispy color or a hexadecimal RGB color representation.
+        If a tuple, the first value must be a string to assign as a name to a
+        colormap and the second item must be a Colormap. If a dict, the key must
+        be a string to assign as a name to a colormap and the value must be a
         Colormap.
     contrast_limits : list (2,)
-        Color limits to be used for determining the colormap bounds for
-        luminance images. If not passed is calculated as the min and max of
+        Value limits to be used for determining the minimum and maximum colormap bounds for
+        luminance images. If not passed, they will be calculated as the min and max of
```

Comment:
```suggestion
        Intensity value limits to be used for determining the minimum and maximum colormap bounds for
        luminance images. If not passed, they will be calculated as the min and max intensity value of
```

Reply:
Can you also suggest for the matching viewer_model and view_layer docstrings?
Otherwise I will tomorrow.

Reply:
done